### DISCUSSION TITLE
How to disable form required validation

### DISCUSSION THREAD

**Discussion by invaders-xx** [filamentphp/filament#3192] [Help]
I have a case where a form can be saved as draft, and I would like to disable validation of required fields. Is there any tricks to implement such logic ? Many thanks in advance for your help.
I was thinking of implementing a toggle to specify whether it is draft or not and set required to fields that are supposed to be required but I don't know how to tap inputs in a closure

**Comment by madsem** [comment]
You could use `$get` I think.

Like:
```php
->required(fn (Closure $get) => $get('draft') === false),
```

**Comment by moeezali2375** [comment]
```php
$state = $this->form->getRawState();
```
getRawState() will ignore all the form validations.
